admission_control/admission_control_service/Cargo.toml

```diff
@@ -36,3 +40,7 @@ types = { path = "../../types", features = ["testing"] }
 
 [build-dependencies]
 build_helpers = { path = "../../common/build_helpers" }
+
+[features]
+default = []
+fuzzing = ["storage_service", "proptest_helpers", "proptest"]
```

Comment:
Needs a newline at the end

Reply:
[interesting](https://thoughtbot.com/blog/no-newline-at-end-of-file), I was wondering why github complained about that.

Reply:
we should have a lint for this tbh.

Reply:
@bmwill  :P ?